### PR TITLE
Translate the word "circle" to a better translation in "prototype inheritance" article

### PR DESCRIPTION
**Pull request by isArman**
correcting some grammatical mistakes.

### CODE REVIEW

File: 1-js/08-prototypes/01-prototype-inheritance/article.md [hunk 1]
```diff
@@ -126,7 +126,7 @@ alert(longEar.jumps); // true (rabbit از)
 
 فقط دو محدودیت وجود دارد:
 
-1. مرجع‌ها نمی‌توانند درون دایره قرار بگیرند. اگر ما تلاش کنیم که `__proto__` را درون یک دایره مقداردهی کنیم، جاوااسکریپت ارور ایجاد می‌کند.
+1. مرجع‌ها نمی‌توانند درون حلقه قرار بگیرند. اگر ما تلاش کنیم که `__proto__` را درون یک حلقه مقداردهی کنیم، جاوااسکریپت ارور ایجاد می‌کند.
```
Comment (mahdiHash) [inline 1-js/08-prototypes/01-prototype-inheritance/article.md:129]: I don't think "حلقه" is a good translation here. Maybe "چرخه" is a better option?
Let's have @mahdyar's thought on this, too.

Comment (mahdyar) [reply 1-js/08-prototypes/01-prototype-inheritance/article.md:129]: I agree.
Loops are another concept in programming languages and may mislead readers.
The original word here was "circle" and "چرخه" is an appropriate translation for it.
```suggestion
1. مرجع‌ها نمی‌توانند درون چرخه قرار بگیرند. اگر ما تلاش کنیم که `__proto__` را درون یک حلقه مقداردهی کنیم، جاوااسکریپت ارور ایجاد می‌کند.
```

**Review by mahdyar — CHANGES_REQUESTED** [review]
Please try to choose an eloquent name for your PR too.

**Comment by isArman** [comment]
I agree. I think "چرخه" is better translation than "حلقه". 
Thank you.

**Comment by mahdiHash** [comment]
@isArman, you don't have to close the PR.
Please reopen it and just commit the suggested changes. You can do this directly from "Files changed" tab below this PR's name.
Your username will be on the contributors list.

**Review by mahdyar — APPROVED** [review]
Thank you, @isArman!

**Pull request merged**